src/common/controllers/ads-controller.js

```diff
@@ -339,6 +367,39 @@ class AdsController extends FakeEventTarget implements IAdsController {
     });
   }
 
+  _handleReturnToLive(adBreaks: Array<RunTimeAdBreakObject>) {
+    this._liveEventManager.listenOnce(this._player, AdEventType.AD_ERROR, () => {
+      this._pushNextAdsForLive(adBreaks);
+      this._liveEventManager.removeAll();
```

Comment:
export these 2 lines to function and use below as well

Reply:
done